migrations/versions/c02a75d4f418_.py

```diff
@@ -0,0 +1,160 @@
+"""empty message
```

Comment:
Should replace this with a message describing this initial migration as matching the previous state of the database.